---a/net/minecraft/item/ItemDoublePlant.java
+++b/net/minecraft/item/ItemDoublePlant.java
@@ -1,18 +1,13 @@
 package net.minecraft.item;
 
-import com.google.common.base.Function;
 import net.minecraft.block.Block;
-import net.minecraft.block.BlockDoublePlant;
-import net.minecraft.world.ColorizerGrass;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ItemDoublePlant extends ItemMultiTexture
-{
+import com.google.common.base.Function;
+
+public class ItemDoublePlant extends ItemMultiTexture {
+
     private static final String __OBFID = "CL_00000021";
 
-    public ItemDoublePlant(Block block, Block block2, Function nameFunction)
-    {
+    public ItemDoublePlant(Block block, Block block2, Function nameFunction) {
         super(block, block2, nameFunction);
     }
-
-    @SideOnly(Side.CLIENT)
-    public int getColorFromItemStack(ItemStack stack, int renderPass)
-    {
-        BlockDoublePlant.EnumPlantType enumplanttype = BlockDoublePlant.EnumPlantType.byMetadata(stack.getMetadata());
-        return enumplanttype != BlockDoublePlant.EnumPlantType.GRASS && enumplanttype != BlockDoublePlant.EnumPlantType.FERN ? super.getColorFromItemStack(stack, renderPass) : ColorizerGrass.getGrassColor(0.5D, 1.0D);
-    }
-}
+}
